app/uk/gov/hmrc/taxenrolmentassignmentfrontend/models/PTEnrolmentOnOtherAccount.scala

```diff
@@ -19,5 +19,6 @@ package uk.gov.hmrc.taxenrolmentassignmentfrontend.models
 case class PTEnrolmentOnOtherAccount(
   currentAccountDetails: AccountDetails,
   ptAccountDetails: AccountDetails,
-  saUserCred: Option[String]
+  saUserCred: Option[String],
+  currentAccountHasMTDIT: Boolean
```

Comment:
why is currentAccountHasMTDIT not in AccountDetails? AccountDetails has an hasSA field already.